### PR TITLE
Add SSSCI wrapping with Python test and baseline

### PR DESCRIPTION
**Pull request by dzenanz**

### CODE REVIEW

**Review by tbirdso — COMMENTED** [review]
Looks good so far. @dzenanz please ping me when this is ready for formal review.

**Comment by dzenanz** [comment]
This is ready for review.

File: wrapping/test/CMakeLists.txt [hunk 1]
```diff
@@ -58,6 +58,21 @@ itk_python_add_test(NAME PythonCurvilinearResampleTest
...
+  )
+
+itk_python_add_test(NAME PythonSliceSeriesResampleTest
+  TEST_DRIVER_ARGS
+  --compareIntensityTolerance 1
+  --compare
+    DATA{${test_baseline_dir}/itkSliceSeriesSpecialCoordinatesImageTestPython.mha}
+    ${ITK_TEST_OUTPUT_DIR}/itkSliceSeriesSpecialCoordinatesImageTestPython.mha
+  COMMAND PythonSliceSeriesResample.py
+    -i DATA{${test_input_dir}/bmode_p59.hdf5}
```
Comment (dzenanz) [inline wrapping/test/CMakeLists.txt:72]: I don't have HDFView installed, so I can't check whether this file contains per-slice transforms. Do you have it Tom?

Comment (tbirdso) [reply wrapping/test/CMakeLists.txt:72]: I do not.

Comment (dzenanz) [reply wrapping/test/CMakeLists.txt:72]: I installed it now from [official site](https://www.hdfgroup.org/downloads/hdfview/), and I can see that it contains extra information.
bmode_p95_hdf5

Comment (tbirdso) [reply wrapping/test/CMakeLists.txt:72]: I'm unclear, are there transforms within the file? Based on component names it appears to be a rectangular B-Mode image with some metadata for collection angles.

Comment (tbirdso) [reply wrapping/test/CMakeLists.txt:72]: I'm not sure I fully understand, does that mean that this is a valid test image? Or not valid?

Comment (dzenanz) [reply wrapping/test/CMakeLists.txt:72]: The stuff read is shoved into MetaDataDictionary:
https://github.com/KitwareMedical/ITKUltrasound/blob/v0.6.2/src/itkHDF5UltrasoundImageIO.cxx#L394-L414

I converted this PR into draft until I figure out what to do with it.

Comment (dzenanz) [reply wrapping/test/CMakeLists.txt:72]: Only rotation seems to be respected. This is probably easiest to debug by modifying https://github.com/KitwareMedical/ITKUltrasound/blob/master/test/itkSliceSeriesSpecialCoordinatesImageTest.cxx to use `bmode_p59.hdf5` and set the metadata similarly to https://github.com/dzenanz/ITKUltrasound/commit/6ed1737902ed9a53bb870f767de105a01678e1aa

Tom, if you have time to do it, that would be good. I am going on vacation the next two weeks. And I  will be mostly working on a different project today and tomorrow (with pending progress update tomorrow afternoon).

Comment (dzenanz) [reply wrapping/test/CMakeLists.txt:72]: @tao558 might pitch in. Coordinate with him while I am away.

Comment (dzenanz) [reply wrapping/test/CMakeLists.txt:72]: I found a solution. Problem was missing wrapping for `UltrasoundImageFileReader`. That should be used by the test. I will push the update soon.

**Comment by dzenanz** [comment]
I will squash some commits and reorder, to have less back and forth.

**Review by thewtex — APPROVED** [review]
Very nicely done! :100:

**Comment by dzenanz** [comment]
I will merge and tag once the CI finishes (probably late today or tomorrow).

**Pull request merged**